src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Event } from "./event/event";
 
 function App() {
   return (
-    <div className="App">
+    <div className="App min-vh-100 d-flex flex-column align-items-center">
```

Comment:
Have a look here https://react-bootstrap.github.io/docs/layout/grid for how the `<Container>` component works. You can still use classes, but using the components is the "react way". I believe you can still achieve what you're after using the components. It also results in much more "responsive" layouts, since many of the things you need are built in.

Additionally, I don't think we need to set the min view height, the events themselves will determine the vertical size of the root component. What is the `App` class referencing?

<details><summary>spoiler</summary>
Consider something like this, that starts with a container, then works with the rows and columns to create the desired layout.

```tsx
function App() {
  return (
    <Container>
      <Row className="text-center">
        <Col md={3}></Col>
        <Col md={6}>
          <header>
            <h1>Eventual</h1>
          </header>
        </Col>
        <Col md={3}></Col>
      </Row>
      <Row>
        <Col md={3}></Col>
        <Col md={6}>
          <Event
            title="Bike Ride"
            description="Nice ride along the coast"
            start={new Date()}
            end={new Date()}
            url="https://www.google.com"
            email="hello@world.com"
          />
        </Col>
        <Col md={3}></Col>
      </Row>
    </Container>
  );
}
```

</details>